toothpick-sample/src/main/java/com/example/toothpick/activity/SimpleBackpackItemsActivity.kt

```diff
@@ -0,0 +1,100 @@
+package com.example.toothpick.activity
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.toothpick.R
+import com.example.toothpick.activity.AddNewActivity.NEW_ITEM_NAME_KEY
+import com.example.toothpick.adapter.BackpackAdapter
+import com.example.toothpick.adapter.IBackpackAdapter
+import com.example.toothpick.annotation.ApplicationScope
+import com.example.toothpick.helper.NotificationHelper
+import com.example.toothpick.model.Backpack
+import toothpick.ktp.KTP
+import toothpick.ktp.binding.bind
+import toothpick.ktp.binding.module
+import toothpick.ktp.binding.toClass
+import toothpick.ktp.delegate.inject
+import toothpick.ktp.delegate.lazy
+import toothpick.smoothie.lifecycle.closeOnDestroy
+
+
+/**
+ * Simple version of the BackpackItemsActivity.
+ *
+ * Here we do not retain the backpack at all.
```

Comment:
In this app, the backpack is not retained on configuration changes as it belongs to the activity scope which follows the lifecycle of activity instances: when an instance is destroyed, its associated scope is closed and a new scope with a new backpack will be created when the new instance of the activity is created.